tools/hack/helm-readme-gen.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+gitroot=$(git rev-parse --show-toplevel)
+#Install latest helm-docs cli
+go install github.com/norwoodj/helm-docs/cmd/helm-docs@latest
```

Comment:
normally, eg does not use this way, please refer `tools.mk`.
BTW, you should pin to a stable version rather than `latest`.